### DISCUSSION TITLE
Lession 1 Getting ether from Rinkeby

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#1558] [Q&A]
Hi I can not ether from rinkeby by tweeting. It says insufficient funds for gas prices and values. Please help

**Comment by cromewar** [comment]
Hello @slklk The Rinkeby Faucet tends to go down time to time, I strongly recommend chainlink Faucet:

https://faucets.chain.link/
